tasks/compare-compiled-output.js

```diff
@@ -371,13 +363,34 @@ async function main(
 
 	cleanAndMkdir(pathing.latest);
 
+	const promises = [];
```

Comment:
This moves the rendering assets to the .diff-output folder so it can be hosted together on Netlify.